src/titlepane/README.md

```diff
@@ -0,0 +1,59 @@
+# @dojo/widgets/titlepane/TitlePane widget
+
+Dojo 2's `TitlePane` component can be used to display content inside a window with a titlebar. When the titlebar is clicked, the content collapses. This component provides default styling for the titlebar, and content area, and will respond to different screen sizes responsively.
+
+## Features
+
+- Content can be prevented from collapsing
+
+![Image of basic title pane](http://placekitten.com/450/300)
+
+### Keyboard Usage
+
+`TitlePane` supports standard keyboard navigation for toggling the content open or closed.
+
+**TitleBar Events**:
+
+- Space bar: toggles the pane content of a closeable `TitlePane`
+
+### Accessibility Features
+
+- The titlebar and aria-labelledby have screen-reader-accessible labels and instructions
+- The text within a closeable titlebar is contained within a `<button>` with screen-reader-accessible instructive text
```

Comment:
I don't think this is entirely accurate, since we're using `role="button"` and not `<button>` (I'm still not entirely sure why we don't just use `<button>`, actually). If I were to write the a11y features, I might do something like this:
```markdown
- The titlebar has a role of heading and accepts a `headingLevel` property to set the correct heading level
- The accordion panel has `aria-labelledby` set to the titlebar
- The accordion opens on Enter and Space
- `aria-expanded` is set to `true` or `false` on the toggle button depending on `properties.open`
- `aria-controls` on the toggle button points to the accordion content
```